file/sst_file_manager_impl.cc

```diff
@@ -256,7 +256,7 @@ void SstFileManagerImpl::ClearError() {
   while (true) {
```

Comment:
If this can be reproduced, we can try to add a unit test with a sync point here to reproduce this sequence.

Reply:
Hmm... this is hard since this happens in test cleaning up phase. If it is too hard, then forget about it. We can have it here and see whether the test failure still h appends in the future.

Reply:
Yeah, I couldn't figure out a repro using sync points. Ended up with sleeps in the SstFileManagerImpl destructor and ClearError to repro.